controllers/components/index/login-form.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+
+import regex from '../helpers/regex';
+
+export default class LoginForm extends React.Component {
+    constructor( props ) {
+        super( props );
+
+        this.state = {
+            submitButton: 'Login',
+            email: '',
+            password: '',
+            errorMessage: '',
+            formIsValide: '',
+            displayError: ''
+        };
+
+        this._checkEmptyFields = this._checkEmptyFields.bind( this );

```

Comment:
This doesn't look right, what are you trying to fix ?

Reply:
Nothing, forgot to delete this, my bad.